src/app/public/slide.ts

```diff
@@ -26,3 +26,26 @@ export const skyAnimationSlide = trigger('skyAnimationSlide', [
     animate('150ms ease-in')
   )
 ]) as AnimationEntryMetadata;
+
+export const skyAnimationSlideNoTrim = trigger('skyAnimationSlideNoTrim', [
```

Comment:
@Blackbaud-TrevorBurch would you mind adding some documentation here to denote what makes this one unique over its predecessor?